index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple Addition Program</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Simple Addition Program</h1>
+        <div class="info">
+            <p class="text-info">Input an operator by selecting any of the following alphabets below to perform an addition operation.
+                <br>
+                <br>
+                a , b , c , d   
```

Comment:
Since you are creating a calculator for only addition i think just one option will do.